Bugzilla/WebService/Product.pm

```diff
@@ -134,6 +135,7 @@ sub get {
     }
 
     # Now create a result entry for each.
+    local %FLAG_CACHE = ();
```

Comment:
Could we just do my %FLAG_CACHE here and not need the outer our %FLAG_CACHE? Seems like it would give the same benefit.

Reply:
I don't think you can localize an entire lexical hash, so it has to be a package var.
Lookup time is slower but it should still be faster overall.